tests/Oryx.Tests.Common/PortHelper.cs

```diff
@@ -0,0 +1,23 @@
+﻿// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
+
+using System.Net;
+using System.Net.Sockets;
+
+namespace Oryx.Tests.Common
+{
+    public static class PortHelper
+    {
+        public static int GetNextAvailablePort()
```

Comment:
@arroyc This should help you in your quest for more parallel tests 😄 ...I had tested this in my Windows and Linux machines and it worked fine.

Reply:
@kichalla  … yepp experiment is on https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=2669129  😄